html/changelogs/listerla-gradients.yml

```
@@ -0,0 +1,5 @@
+author: listerla
+
+delete-after: True
+
+  - imageadd: "Ported 'Reflected', 'Reflected (Inverse), and 'Wavy' hair gradients from World Server."
```

```suggestion
  - imageadd: "Ported \"Reflected\", \"Reflected (Inverse)\", and \"Wavy\" hair gradients from World Server."
```
The changelog system fails, most likely due to issues with the quotes